src/Nest/Mapping/Types/Core/Text/TextProperty.cs

```diff
@@ -3,59 +3,58 @@
 using System.Runtime.Serialization;
 using Elasticsearch.Net.Utf8Json;
 
-namespace Nest
-{
+namespace Nest {
```

Comment:
The editorconfig file should format block braces to separate lines with `CTRL + E` `CTRL + F`

Reply:
Yes, I saw this. I think VS 2019 is not happy about some format in the .editorconfig, which causes formatting to actually make things worse. I’ve now fixed the .editorconfig locally. If there is interest I can submit a new PR.

Reply:
Please feel free to open a PR for the editorconfig 👍